modules/mediawiki/manifests/init.pp

```diff
@@ -13,6 +12,8 @@
     include mediawiki::logging
     include mediawiki::php
     include mediawiki::monitoring
+    include mediawiki::firejail
+    include mediawiki::deploy
```

Comment:
Make only include if staging is turned on?

Reply:
There are checks within but we can't check here as the file also includes `mediawiki::remote_sync` stuff which is not on when `mediawiki::use_staging` is

Reply:
Remote sync doesn't work without staging

Reply:
Oh wait I see

Reply:
We should probably only put canary + staging in the deploy file and have it for stuff needed on a deploy server. To be honest, remote_sync could probably be changed to and else statement as the only places it is not on are the deploy servers.

Reply:
What's the difference between staging and canary, can't they be merged also?

Reply:
No, canary is what allows syncing to remotes so not enabled on test even though test has staging.

Reply:
Done